src/ripple/app/misc/FeeVoteImpl.cpp

```diff
@@ -228,7 +228,7 @@ FeeVoteImpl::doVoting (Ledger::ref lastClosedLedger,
         Serializer s;
         trans.add (s, true);
 
-        SHAMapItem::pointer tItem = std::make_shared<SHAMapItem> (
+        std::shared_ptr<SHAMapItem> tItem = std::make_shared<SHAMapItem> (
```

Comment:
I think it was more of a search and replace kind of change rather than deliberating over each line manually